builtin/pack/builder.go

```diff
@@ -352,7 +352,7 @@ func (b *Builder) Build(
 	if err != nil {
 		return nil, err
 	}
-	if serverInfo.Architecture != "amd64" {
+	if serverInfo.Architecture == "aarch64" || serverInfo.Architecture == "arm64" {
```

Comment:
Probably ok unless someone is running buildpack on MIPS? RISCV? Then they won't get the warning.

Reply:
Hrm that's fair. Maybe it should be `... != "amd64" || ... != "x86_64"`?

Reply:
Updated as such!